src/app/storage/query/middlewares.ts

```diff
@@ -0,0 +1,45 @@
+import { Method } from 'axios';
+import { NextFunction, Request, Response } from 'express';
+import { fetchQueries, getStoredQueryById, methodRequiresQueryId } from './utils';
+
+export const endpointRequiresQueryId = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
abstracted common/shared endpoint middlewares into a separate file, for separation of concerns.